test/FunctionalTests/Microsoft.OData.Core.Tests/Json/JsonWriterAsyncBaseTests.cs

```diff
@@ -231,7 +231,7 @@ public async Task WritesMixedObjectWithRawValuesCorrectlyAsync()
         }
 
 #if NETCOREAPP
-        [Fact]
+        [Fact(Skip = "This tests fails in release pipeline and locally but passes in build pipeline. Needs to be investigated.")]

```

Comment:
Link to issue? Issue should be part of ODL 8 work items so we don't ship a bug in case that's the reason the test is failing

Reply:
Re-enabled this test.